stable/kubecost/values.yaml

```diff
@@ -87,13 +87,19 @@ cost-analyzer:
       # This overrides the cluster_id set in prometheus.server.global.external_labels.
       # NOTE: This does not affect the external_labels set in prometheus config.
       clusterIDConfigmap: kubecost-cluster-info-configmap
+      image:
+        tag: v2.19.1
+      extraFlags:
+      - web.enable-admin-api
+      - web.enable-lifecycle
+      - storage.tsdb.wal-compression
       resources:
         limits:
+          cpu: 2000m
+          memory: 5000Mi
```

Comment:
why do you set 5G ? can you justify this change ?

Reply:
Only Prometheus sets this limits 🤔 ... is this consuming that many resources ? I have been looking at the usage and it cannot be compared with the native prometheus used by the whole cluster

This shows how during the last 30d the memory did go beyond  1.4G and cpu beyond 0.1. I'd suggest yo lower a bit this values accordingly to a common usage. I think the initial values are good enough.
![Screen Shot 2020-06-25 at 11 28 52 AM](https://user-images.githubusercontent.com/3602792/85694124-239df600-b6d7-11ea-8b21-2279f5491bcc.png)

```
cpu: 1000m
          memory: 2500Mi
```